src/get-started/first-java-service.md

```diff
@@ -0,0 +1,615 @@
+# First Exonum Java Service Tutorial
+
+<!-- cspell:ignore Lorean, Intelli, testnet -->
+
+This tutorial is aimed at beginners in Exonum Java. It is an introduction
+into service development in Java. As such, it does not cover some important
+to real-world services topics, like authorization, or proofs of authenticity
+for service data, or testing, but gives a foundation to learn about that
+in subsequent materials.
+
+<!-- todo: Is it the right introductory page on main Exonum principles and
+  abstractions? -->
+It is recommended to read an [Exonum design overview](design-overview.md)
+before proceeding with this tutorial.
+
+The full tutorial code is available in our Git repository.
+
+## Prerequisites
+
+The following software must be installed:
+
+- [JDK 11+](https://jdk.java.net/)
+- [Apache Maven 3.6+](https://maven.apache.org/install.html)
+- [Python 3.6+](https://www.python.org/downloads/)
+- [Exonum Java 0.10](./java-binding.md#installation)
+- [cURL](https://curl.haxx.se/download.html).
+
+## Service Overview
+
+Blockchains are often used to implement secure registries. In this tutorial,
+we will implement a vehicle registry. The registry will keep a record of
+vehicles; and allow to create new vehicles, and change their owner.
+It will also provide REST API to query the registry.
+
+## Service Implementation
+
+### 1 Creating a Project
+
+#### Generate from Template
+
+First, create a new service project with a Maven archetype.
+
+```shell
+mvn archetype:generate \
+    -DarchetypeGroupId=com.exonum.binding \
+    -DarchetypeArtifactId=exonum-java-binding-service-archetype \
+    -DarchetypeVersion=0.10.0
+```
+
+<!--
+todo: Or replace with a non-interactive command, so that we don't rely
+on user input?
+-->
+When prompted, enter the values of project properties, for example:
+
+- `com.example.car` for `groupId`
+- `car-registry` for `artifactId`
+- `1.0.0-SNAPSHOT` for `version`
+
+Keep the default for `package` and confirm the input.
+
+Verify that the project has been generated correctly and the dependencies
+are installed by running its integration tests:
+
+```shell
+cd car-registry/
+mvn verify
+```
+
+You shall see in the output that `com.example.car.MyServiceIntegrationTest`
+completed successfully and the build passes.
+
+??? fail "Getting a build error?"
+    - If you get `java.lang.LinkageError` in the test, check that Exonum Java
```

Comment:
Nit: I'd remove top-level list formatting here and instead format two list items as two paragraphs.